benchmarks/src/main/scala/zio/redis/hash/HGetAllBenchmarks.scala

```diff
@@ -51,5 +51,5 @@ class HGetAllBenchmarks extends BenchmarkRuntime {
   }
 
   @Benchmark
-  def zio(): Unit = zioUnsafeRun(ZIO.foreach_(items)(_ => hGetAll[String, String, String](key)))
+  def zio(): Unit = zioUnsafeRun(ZIO.foreach_(items)(_ => hGetAll[String](key).returning[String, String]))
```

Comment:
Is it possible to remove the type remaining type as well, e.g.:

```suggestion
  def zio(): Unit = zioUnsafeRun(ZIO.foreach_(items)(_ => hGetAll(key).returning[String, String]))
```